src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -279,7 +279,7 @@ def _move_table(
 
             return True
         except NotFound as err:
-            if "[TABLE_OR_VIEW_NOT_FOUND]" in str(err):
+            if "[TABLE_OR_VIEW_NOT_FOUND]" in str(err) or "[DELTA_TABLE_NOT_FOUND]" in str(err):
                 logger.error(f"Could not find table {from_table_name}. Table not found.")
             else:
                 logger.error(err)
```

Comment:
This logger.error needs context (the table name)

Reply:
added